source/runtime.cpp

```diff
@@ -1022,7 +1039,7 @@ void reshade::runtime::save_current_preset() const
 	save_preset(_current_preset_path);
 }
 
-void reshade::runtime::save_screenshot()
+void reshade::runtime::save_screenshot(bool before)
```

Comment:
I'd change this to accept a postfix string which is appened to the path name instead of a boolean:
```c++
void reshade::runtime::save_screenshot(const std::string &postfix)
{
   ...
   const std::wstring screenshot_path = least + postfix + (_screenshot_format == 0 ? L".bmp" : L".png");
   ...
}
```